tests/openmetrics/test_exposition.py

```diff
@@ -94,6 +95,125 @@ def test_histogram(self):
 # EOF
 """, generate_latest(self.registry))
 
+
+    def test_native_histogram(self):
```

Comment:
```suggestion
    def test_native_histogram(self) -> None:
```
To get type checks. Also for all the other tests here.